src/utils/assets.js

```diff
@@ -137,7 +137,7 @@ export const getFormattedRate = (
 export const calculateMaxAmount = (
   token: string,
   balance: number | string,
-  txFeeInWei: BigNumber,
+  txFeeInWei: ?(BigNumber | string | number),
```

Comment:
We have a type `Value` for this in `utils/common`

Reply:
Fixed